ucum-java8/src/main/java/systems/uom/ucum/UCUM.java

```diff
@@ -768,9 +786,116 @@ public static UCUM getInstance() {
 	/** As per <a href="http://unitsofmeasure.org/">UCUM</a> standard. */
 	public static final Unit<Power> HORSEPOWER = addUnit(new ProductUnit<Power>(
 			FOOT_INTERNATIONAL.multiply(POUND_FORCE).divide(SECOND)));
-	// ///////////////////////////////////////////////////////
-	// SECTIONS §44-§46 skipped; implement later if needed //
-	// ///////////////////////////////////////////////////////
+	
+	////////////////////////////////////////////
```

Comment:
Honestly no idea where it first came up. It is not in JScience 5, so could be a formatting tool trying to expand or reduce the number of characters. This can be fixed wherever you spot it. Already did for most of the other unit systems like SI or implementation classes ;-)